shared/macros-bash.jinja

```diff
@@ -101,41 +101,40 @@ LC_ALL=C sed -i "/{{{ regex }}}/{{{ modifier }}}" "{{{ path }}}"
     {{%- else -%}}
         {{%- set grep_args="-q -m 1" -%}}
     {{%- endif -%}}
-    {{% if (insert_after == "" and insert_before == "")  or insert_after == "EOF" -%}}
-# Insert at the end of the file
 cp "{{{ path }}}" "{{{ path }}}.bak"
+    {{% if not (insert_after or insert_before)  or insert_after == "EOF" -%}}
+# Insert at the end of the file
```

Comment:
This comment is shifted by 4 spaces to the right, probably because of 4 spaces before the if on previous line.  That breaks our beautiful indentation of the generated code. Adding `-` after `{{%` won't help because that would merge the comment with the `cp` command. Could you take a look into the indentation?

Reply:
Good, I hope that I have fixed that.

Reply:
I have checked that it doesn't occur anymore.